### PR TITLE
Fixes multiple PRs raise by test team

### LINKED ISSUE #93: salt execution module junos.rollback CLI example syntax is not correct (salt 'device_name' junos.rollback 10)
### Description of Issue/Question

Documentation issue:

https://github.com/Juniper/salt/blob/0148fe0de1697c803747aa15f36376daeb6a008b/salt/modules/junos.py#L423

Should be id =10

### PR DESCRIPTION
**Pull request by rsmekala**
### What does this PR do?
Fixes #93 #94

### CODE REVIEW

**Comment by rsmekala** [comment]
Fixes #96

**Comment by rsmekala** [comment]
Fixes #98

**Comment by rsmekala** [comment]
Fixes #99

**Comment by rsmekala** [comment]
Fixes #100

**Comment by rsmekala** [comment]
Fixes #103

**Comment by rsmekala** [comment]
Fixes #105 

New behavior:
```
➜ salt "vmx" junos.install_config salt://config.set mode=exclusive comment="testing1" update=True

vmx:
    ----------
    message:
        configuration database locked by:
          regress terminal pts/0 (pid 92400) on since 2019-04-12 14:56:31 IST, idle 00:04:17
              exclusive [edit]
    out:
        False


```

File: salt/modules/junos.py [hunk 1]
```diff
@@ -216,15 +216,15 @@ def rpc(cmd=None, dest=None, **kwargs):
...
+    dest = op.pop('dest', None)
 
     if cmd in ['get-config', 'get_config']:
         filter_reply = None
         if 'filter' in op:
             try:
                 filter_reply = etree.XML(op['filter'])
             except etree.XMLSyntaxError as ex:
-                ret['message'] = "Invalid filter " + ex.__repr__()
+                ret['message'] = "Invalid filter " + str(ex)
```
Comment (vnitinv) [inline salt/modules/junos.py]: use .format

File: salt/modules/junos.py [hunk 2]
```diff
@@ -999,7 +1020,7 @@ def zeroize():
     ret = {}
     ret['out'] = True
     try:
-        conn.cli('request system zeroize')
+        conn.rpc.request_system_zeroize()
```
Comment (vnitinv) [inline salt/modules/junos.py]: can you checck if this rpc is there atleast on 14.x

**Review by vnitinv — CHANGES_REQUESTED** [review]
please do the minor changes requested.

File: salt/modules/junos.py (hunk 2 above)
Comment (rsmekala) [reply salt/modules/junos.py]: Removed the change from this PR. Will confirm from test and replace it in next PR if possible.

File: salt/modules/junos.py (hunk 1 above)
Comment (rsmekala) [reply salt/modules/junos.py]: Done

**Review by vnitinv — APPROVED** [review]

**Pull request merged**